lib/pulse/lib/Pulse.Lib.ConditionVar.fst

```diff
@@ -286,7 +288,8 @@ ensures
             (predicate_at b.core.tab 0.5R preds')
             0 i (Seq.length preds');
         fold (istar preds');
-        fold (maybe_holds v q preds'); 
+        rewrite (istar preds') as (maybe_holds v q preds');
+        // fold (maybe_holds v q preds');
```

Comment:
Why does this no longer work?

Reply:
To answer my own question, it's because `maybe_holds` is a match.  And so you need to tell the matcher which branch is true.